spec/api/spec.rb

```diff
@@ -0,0 +1,97 @@
+module Spec
+
+  ###############
+  # Models
+
+  def patient_spec
+    { type:       :object,
+      properties: { id:               { type: :string, format: :uuid },
```

Comment:
Let's put hash keys on a new line after the `{` so we don't have so much left whitespace